CHANGELOG.md

```diff
@@ -8,6 +8,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 <!-- next-header -->
 ## [Unreleased] - ReleaseDate
 
+## [0.1.0] - 2023-07-22
+
+### Added
+
+- Added `Serialize` and `Default` traits to `AdvisoryStatus` and `Advisories`
+- Added `Deserialize` trait to `Advisories`
+- Added `Ord` trait to `AdvisoryStatus`
+
```

Comment:
:wrench: I need to get better at updating the changelog when making a changes.
Also this has changed and removed so the release should probably be a '0.2.0' release
```suggestion

### Changed

- `x509` feature has been renamed to `mbedtls`
- `UnverifiedCertChain` is no no longer public. One should use the `MbedTlsCertificateChainVerifier` instead.
- `CertificateREvocationList` is no longer public. One should use the `MbedTlsCertificateChainVerifier` instead.
- `EvidenceVerifier` will now also verify the signature of the `Quote3`
- `Quote3Verifier::new()` now takes an `Option<VerifyingKey>`, instead of a `VerifyingKey`.

### Removed

- `VerifiedCertChain` has been removed. One should sue the `MbedTlsCertificateChainVerifier` instead.

```